docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -14,6 +14,15 @@ Use Package Source Mapping along side other [best practices](..\concepts\Securit
 
 Starting with [NuGet 6.0](..\release-notes\NuGet-6.0.md), you can centrally declare which source each package in your solution should restore from in your nuget.config file.
 
+Starting with Visual Studio 17.5, you can add and remove Package Source Mappings with the Visual Studio Options Dialog
+
+### Visual Studio support
+| Visual Studio | Package Source Mapping | Support in Tools -> Options | Support in Package Manager UI |
+|-----|---------------------|---------------------|---------------------|
+| 17.0 - 17.4 | ✅ Available | ❌ Not available | ❌ Not available |
+| 17.5 | ✅ Available | ✅ Available | ❌ Not available |
+| 17.7 Preview 3| ✅ Available | ✅ Available | ✅ Status displayed |

```

Comment:
Great table, but I think we should simplify with a note to be consistent with all other docs. See https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management#global-package-references for an example

Fine to keep it for now too

Reply:
Chatted offline - may reduce the table into a couple notes once more features have shipped